changelogs/8.13.asciidoc

```diff
@@ -1,7 +1,13 @@
-[[release-notes-head]]
-== APM version HEAD
+[[release-notes-8.13]]
+== APM version 8.13
 
-https://github.com/elastic/apm-server/compare/8.12\...main[View commits]
+https://github.com/elastic/apm-server/compare/8.12\...8.13[View commits]
+
+* <<release-notes-8.13.0>>
+
+[float]
+[[release-notes-8.13.0]]
+=== APM version 8.13.0
 
 [float]
```

Comment:
could not find the way to remove these if no entries... rather than using something complicated I decided to keep them... so the reviewer will be the ones touching this